demo/send/mailingLists/3c869606-e1d4-4d37-9267-5e38bc6efeb8.json

```diff
@@ -0,0 +1 @@
+{"Campaign":null,"SendToDetails":null,"Content":null,"OptIn":0,"SegmentsCount":0,"LastDayChanges":0,"ActiveSubscribers":2,"MailingList":{"Id":"3c869606-e1d4-4d37-9267-5e38bc6efeb8","MailingListCode":"3050944","Name":"Generated for Email Activity","MailingListName":"Generated for Email Activity","ActiveMemberCount":2,"BouncedMemberCount":0,"RemovedMemberCount":0,"UnsubscribedMemberCount":0,"Status":0,"CreatedBy":"7dc7a9bd0a14427f894a66d8f7a938db","CreatedOn":"/Date(1658175357343-0000)/","UpdatedOn":"/Date(1658841548140-0000)/","DeletedMemberCount":0,"IsSupressionList":false},"Entity":null}
```

Comment:
is this "Generated for Email Activity" really used by the solution and must be serialized here? Should it be renamed to something more meaningful if it is being used?